projects/ngx-intl-tel-input/src/lib/ngx-intl-tel-input.component.ts

```diff
@@ -45,64 +45,66 @@ import { PhoneNumberFormat } from './enums/phone-number-format.enum';
 	],
 })
 export class NgxIntlTelInputComponent implements OnInit, OnChanges {
-	@Input() value = '';
+	@Input() value = "";
 	@Input() preferredCountries: Array<string> = [];
 	@Input() enablePlaceholder = true;
 	@Input() customPlaceholder: string;
 	@Input() numberFormat: PhoneNumberFormat = PhoneNumberFormat.International;
-	@Input() cssClass = 'form-control';
+	@Input() cssClass = "form-control";
 	@Input() onlyCountries: Array<string> = [];
 	@Input() enableAutoCountrySelect = true;
 	@Input() searchCountryFlag = false;
 	@Input() searchCountryField: SearchCountryField[] = [SearchCountryField.All];
-	@Input() searchCountryPlaceholder = 'Search Country';
-	@Input() maxLength = '';
+	@Input() searchCountryPlaceholder = "Search Country";
+	@Input() maxLength = "";
 	@Input() selectFirstCountry = true;
 	@Input() selectedCountryISO: CountryISO;
 	@Input() phoneValidation = true;
-	@Input() inputId = 'phone';
+	@Input() inputId = "phone";
 	@Input() separateDialCode = false;
+	@Input() searchCountryPopupContainer: string;
+	@Input() searchCountryPopupPlacement: string = "bottom left";
 	separateDialCodeClass: string;
 
 	@Output() readonly countryChange = new EventEmitter<Country>();
 
 	selectedCountry: Country = {
 		areaCodes: undefined,
-		dialCode: '',
-		htmlId: '',
-		flagClass: '',
-		iso2: '',
-		name: '',
-		placeHolder: '',
+		dialCode: "",
```

Comment:
Many personal code style choices being applied, are these linting rules active in the library?